docroot/profiles/custom/cgov_site/modules/custom/cgov_site_section/cgov_site_section.module

```diff
@@ -38,6 +39,43 @@ function cgov_site_section_entity_bundle_field_info(EntityTypeInterface $entity_
   }
 }
 
+/**
+ * Implements term_cgov_site_sections_parent_validate(), validate default root as parent.
+ */
+function term_cgov_site_sections_parent_validate(&$form, \Drupal\Core\Form\FormStateInterface $form_state) {
+   $langcode =  $form_state->getValue('langcode');
+   $tid =  $form_state->getValue('tid');
+   $vid = 'cgov_site_sections';
+    $terms =\Drupal::entityTypeManager()->getStorage('taxonomy_term')->loadTree($vid);
+    $enParentTermCount = $esParentTermCount = 0;
+    foreach ($terms as $term) {
+      if($term->depth == 0 && $term->langcode == 'en' && $tid != $term->tid) {
```

Comment:
Can you please do this so it is not ONLY Spanish and English? I.E. it should allow for any number of languages, or it should at least allow for those languages specified.

(We had to add Chinese and Portuguese to our old system (and yes, it was never used) and code like this added many hours to that project)